ideascube/serveradmin/tests/conftest.py

```diff
@@ -1,8 +1,23 @@
 import pytest
+import os
 
 from ..backup import Backup
 
 DATA_ROOT = 'ideascube/serveradmin/tests/data/backup'
+BACKUPS_ROOT = 'ideascube/serveradmin/tests/backups'
```

Comment:
Agree with you.
BACKUPS_ROOT come from test_backup.py.
I copy it just to be able to create the backup_root fixture.

However, this should not be necessary with the storage_dir fixture who include also the creation of a new BACKUP_ROOT.

There is lot to do on this point; a lots of tests in test_view.py and test_backup.py still monkeypath themselves the Backup.ROOT stuff. We should cleanup all those tests with better fixture.
But this PR is big enough I think. We should clean this is a second step.